ocs_ci/ocs/precheck.py

```diff
@@ -0,0 +1,25 @@
+import logging
```

Comment:
One empty line between imports from diff group. Look at pep8 imports section

Reply:
Hm, I wonder why autopep8 didn't change it for me. Noted.

Reply:
not sure if autopep8 deal with import orders